tensorrt_llm/serve/openai_server.py

```
@@ -324,6 +340,68 @@ async def create_chat_response(
+    async def openai_mm_encoder(self, request: ChatCompletionRequest, raw_request: Request) -> Response:
+
+        async def create_mm_embedding_response(
+                promise: RequestOutput):
+            await promise.aresult()
+            mm_embedding_handle = promise.mm_embedding_handle
+            num_tokens = mm_embedding_handle["tensor_size"][0]
+            # TODO: need to add more info like input_ids, specific_token_ids, mrope, mm_hashes, etc
+            return ChatCompletionResponse(
+                id=str(promise.request_id),
+                model=self.model,
+                choices=[ChatCompletionResponseChoice(index=0, message=ChatMessage(role="assistant", content="dummy"), mm_embedding_handle=mm_embedding_handle, finish_reason="length")],
```

Ruff (0.12.2)

Line too long (185 > 120)

(E501)